karma.conf.js

```diff
@@ -27,13 +27,15 @@ module.exports = function(config) {
 
     files: [
       'tests/unit/**/*.js',
-      'static/js/modules/**/*.js'
+      'static/js/modules/**/*.js',
+      'static/js/vendor/**/*.js'
```

Comment:
@noahmanger knows more about this than I do, but I'm wondering if `tablist` has diverged enough from wherever we got it that we can take it out of `vendor`.